drivers/sensor/st/lsm6dsv16x/lsm6dsv16x.h

```diff
@@ -112,11 +132,15 @@ struct lsm6dsv16x_data {
 #if defined(CONFIG_LSM6DSV16X_TRIGGER_OWN_THREAD)
 	K_KERNEL_STACK_MEMBER(thread_stack, CONFIG_LSM6DSV16X_THREAD_STACK_SIZE);
 	struct k_thread thread;
-	struct k_sem gpio_sem;
+	struct k_sem intr_sem;
```

Comment:
Is this modification part of I3C changes?

Reply:
EDIT: I saw it later on in trigger file. Not sure why the name has been changed, but ok.

Reply:
it was changed because i3c uses the same semaphore. I3C uses the in-band interrups (IBI) which come over the data lines rather than requiring a separate pin through a GPIO. I renamed it to `intr_sem` so it wouldn't be confusing to both I3C and other serial comm users

Reply:
OK, got it. Not very expert on I3C. Thanks for explaining.